data/templates/css/meson.build

```diff
@@ -0,0 +1,29 @@
+# Copyright 2018 Endless Mobile, Inc.
+#
+# We only need to have this file in this subdirectory
+# as outputs to custom_target cannot have path separators
+# their names.
+
+sass = find_program('sassc')
```

Comment:
I left a comment about this unused variable last time, but it doesn't seem to have been addressed, or I'm missing it.

Reply:
I mentioned in my earlier comment that I would actually prefer to use the result of the check. It's possible to have `sassc` installed outside of the path, or have two copies installed. You can get Meson to find the right binary in that case, but the Python script will ignore that and just use whichever one is first in the path. I'd suggest passing the result of this check as the first parameter of the Python script, so that it uses whichever `sassc` you tell it to.

Reply:
Makes sense